test/spec/index.spec.js

```diff
@@ -14,33 +16,147 @@ describe('index.js', function() {
       var parsedOptions = krabby._getOptions(options);
       assert(_.isEqual(options, parsedOptions));
     });
+    it('finds a krabby config when found inside of another object', function() {
```

Comment:
[it tests this thing](https://github.com/patrickkettner/krabby/blob/master/index.js#L29-L31)